.github/workflows/e2ePerf.yml

```diff
@@ -0,0 +1,90 @@
+name: End to End Tests (Performance)
+on:
+  workflow_dispatch:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'phale/perf-poc'
+        type: string
+
+# all possible combinations of OS and architecture supported by yourkit
+env:
+  Linux-ARM: ./yourkit-distro/linux-arm-32/libyjpagent.so
+  Windows-X86: ./yourkit-distro/windows-x86-32/yjpagent.dll
+  Linux-X86: ./yourkit-distro/linux-x86-32/libyjpagent.so
+  Windows-X64: ./yourkit-distro/windows-x86-64/yjpagent.dll
+  Linux-ARM64: ./yourkit-distro/linux-arm-64/libyjpagent.so
+  macOs-X64: ./yourkit-distro/mac/libyjpagent.dylib
+  Windows-ARM64: ./yourkit-distro/windows-arm-64/yjpagent.dll
+  Linux-X64: ./yourkit-distro/linux-x86-64/libyjpagent.so
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest]
+        nodeVersion:
+          - 16.15.1
+        vscodeVersion:
+          - stable
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: ./salesforcedx-vscode
+          ref: ${{ github.event.ref }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.nodeVersion }}
+          cache: npm
+          cache-dependency-path: |
+            salesforcedx-vscode/package-lock.json
+            salesforcedx-vscode-automation-tests/package-lock.json
+      - name: Clone automation tests
+        uses: actions/checkout@v3
+        with:
+          repository: forcedotcom/salesforcedx-vscode-automation-tests
+          path: salesforcedx-vscode-automation-tests
+          ref: ${{ inputs.automationBranch }}
+      - name: Install Extension Dependencies
+        run: |
+          npm install
```

Comment:
I think we also need to run npm run compile here ?

Reply:
Yes - that was a missing step for us previously!